Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail_test.py

```diff
@@ -6,11 +6,34 @@
 from CommonServerPython import *
 from MicrosoftGraphMail import MsGraphClient, build_mail_object, assert_pages, build_folders_path, \
     add_second_to_str_date, list_mails_command, item_result_creator, create_attachment, reply_email_command, \
-    send_email_command, list_attachments_command
+    send_email_command, list_attachments_command, main
 from MicrosoftApiModule import MicrosoftClient
 import demistomock as demisto
 
 
+@pytest.mark.parametrize('params, expected_result', [
+    ({'creds_tenant_id': {'password': '1234'}, 'creds_auth_id': {'password': '1234'}}, 'Key must be provided.'),
+    ({'creds_tenant_id': {'password': '1234'}, 'credentials': {'password': '1234'}}, 'ID must be provided.'),
+    ({'credentials': {'password': '1234'}, 'creds_auth_id': {'password': '1234'}}, 'Token must be provided.')
+])
+def test_params(mocker, params, expected_result):
+    """
+    Given:
+      - Configuration parameters
+    When:
+      - One of the required parameters are missed.
+    Then:
+      - Ensure the exception message as expected.
+    """
```

Comment:
Added cases: `Case 1: ...` (See NeiscoutSigtline for example)

Reply:
Also let's add a test to show that no matter which of the three options is used by the user the relevant params are wprking.